README.md

```diff
@@ -4,40 +4,82 @@
-  <a href="https://docs.zama.ai/tfhe-rs"> 📒 Read documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
+  <a href="https://docs.zama.ai/tfhe-rs"> 📒 Documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
 </p>
 <p align="center">
 <!-- Version badge using shields.io -->
   <a href="https://github.com/zama-ai/tfhe-rs/releases">
     <img src="https://img.shields.io/github/v/release/zama-ai/tfhe-rs?style=flat-square">
   </a>
-  <!-- Link to tutorials badge using shields.io -->
-  <a href="#license">
+  <!-- License badge -->
+  <a href="#-license">
     <img src="https://img.shields.io/badge/License-BSD--3--Clause--Clear-orange?style=flat-square">
   </a>
 <!-- Zama Bounty Program -->
   <a href="https://github.com/zama-ai/bounty-program">
     <img src="https://img.shields.io/badge/Contribute-Zama%20Bounty%20Program-yellow?style=flat-square">
   </a>
 </p>
-<hr/>
 
 
-**TFHE-rs** is a pure Rust implementation of TFHE for boolean and integer
-arithmetics over encrypted data. It includes:
- - a **Rust** API
- - a **C** API
- - and a **client-side WASM** API
 
-**TFHE-rs** is meant for developers and researchers who want full control over
-what they can do with TFHE, while not having to worry about the low level
+## Table of Contents
+- **[About](#about)**
+  - [What is TFHE-rs](#-what-is-tfhe-rs)
+  - [Main features](#-main-features)
+- **[Getting Started](#getting-started)**
+   - [Configuration](#-configuration)
+   - [A simple example](#-a-simple-example)
+- **[Resources](#resources)**
+   - [TFHE deep dive](#-tfhe-deep-dive)
+   - [Tutorials](#-tutorials)
+   - [Documentation](#-documentation)
+- **[Working with THFE-rs](working-with-tfhe-rs)**
+   - [Disclaimers](#-disclaimers)
+   - [Citations](#-citations)
+   - [Contributing](#-contributing)
+   - [License](#-license)
+- **[Support](#support)**
+<br></br>
+
+
+## About
+
+### 🟨 What is TFHE-rs
+
+**TFHE-rs** is a pure Rust implementation of TFHE for boolean and integer arithmetics over encrypted data.
+
+It includes:
+- a **Rust** API
+- a **C** API
+- and a **client-side WASM** API
+
+TFHE-rs is designed for developers and researchers who want full control over
+what they can do with TFHE, while not having to worry about the low-level
 implementation. The goal is to have a stable, simple, high-performance, and
 production-ready library for all the advanced features of TFHE.
+<br></br>
+
+### 🟨 Main features
+
+- **Low-level cryptographic library:** implementing Zama’s variant of TFHE, including programmable bootstrapping
+- **Implementation of the original TFHE boolean API:** used as a drop-in replacement for other TFHE libraries
+- **Short integer API:**  enabling exact, unbounded FHE integer arithmetics with up to 8 bits of message space
+- **Public key encryption:** implementation for TFHE
```

Comment:
this looks odd, the content after the `:` is not really useful

Though we could say we have a Size efficient Public Key encryption

Reply:
nope all good